porcupine/_state.py

```diff
@@ -6,11 +6,13 @@
 import os
 import tkinter
 import types
+from ctypes import windll
```

Comment:
This doesn't work on my computer. I don't use Windows, so importing `windll` fails with `ImportError` for me. You need to use e.g. try/except or `if sys.platform == "win32"`.

Reply:
Added.